components/movies/MovieDetails.bs

```diff
@@ -148,6 +149,7 @@ sub OnScreenShown()
         if isChainValid(m.top.itemContent, "json.mediaStreams")
             SetDefaultAudioTrack(m.top.itemContent.json)
             SetUpAudioOptions(m.top.itemContent.json.mediaStreams)
+            SetUpVideoOptions(m.top.itemContent.json.mediaSources)
```

Comment:
Was this always missing?  Also, isChainValid only looks for "mediaStreams" and not "mediaSources"

Reply:
As far as I can see, yes it's always been this way.